rskj-core/src/test/java/co/rsk/vm/PrecompiledContractAddressTests.java

```diff
@@ -0,0 +1,54 @@
+package co.rsk.vm;
+
+import co.rsk.config.TestSystemProperties;
+import co.rsk.core.RskAddress;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import org.apache.commons.lang3.builder.ToStringExclude;
+import org.ethereum.vm.DataWord;
+import org.ethereum.vm.PrecompiledContracts;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * Created by Sergio Demian Lerner on 12/10/2018.
+ */
+public class PrecompiledContractAddressTests {
+
+    public static final String ECRECOVER_ADDR = "0000000000000000000000000000000000000001";
```

Comment:
How about reusing the constants in the `PrecompiledContracts` class?

Reply:
I want to catch future modifications in that class. If I reuse them, it's probable I won't.